backend/tests/conftest.py

```
@@ -1,12 +1,15 @@
+import functools
 import logging
 import os
-import time
 import subprocess
+import time
 from datetime import datetime, timedelta
+from typing import List, Optional, Union
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)